invenio_previewer/extensions/ipynb.py

```diff
@@ -24,10 +25,12 @@ def render(file):
     """Generate the result HTML."""
     with file.open() as fp:
         content = fp.read()
+    try:
+        notebook = nbformat.reads(content.decode("utf-8"), as_version=4)
+    except nbformat.reader.NotJSONError:
+        return _("Error: Not a json file"), {}
 
-    notebook = nbformat.reads(content.decode("utf-8"), as_version=4)
-
-    html_exporter = HTMLExporter()
+    html_exporter = HTMLExporter(template="lab", embed_images=True)
```

Comment:
This works and you have the plots:
```suggestion
    html_exporter = HTMLExporter(embed_images=True, sanitize_html=True)
```
`theme` is by default `light`, `template_name` is by default `lab`.

`sanitize_html` will automatically sanitize content and you can remove it from the Jinja template. I have tested with an `alert()`. We can add a test to be sure, see: https://github.com/jupyter/nbconvert/blob/main/nbconvert/exporters/tests/test_html.py#L240

Reply:
Current rendering:
![Screenshot 2023-09-12 at 23 16 57](https://github.com/inveniosoftware/invenio-previewer/assets/2089455/4df0f686-1d9b-4124-be9e-04b803da404c)